packages/assets-controllers/src/CurrencyRateController.ts

```diff
@@ -167,6 +167,10 @@ export class CurrencyRateController extends StaticIntervalPollingController<Curr
       const testnetSymbols = Object.values(TESTNET_TICKER_SYMBOLS);
       const nativeCurrenciesToFetch = nativeCurrencies.reduce(
         (acc, nativeCurrency) => {
+          if (!nativeCurrency) {
+            return acc;
+          }
```

Comment:
By filtering falsy values here, we stop the error that occurs when calling `undefined.toLowerCase()` as well as avoid passing those values to the API.

Reply:
I'm really not a fan of lying types.
Is it worth updating the function interface to `nativeCurrencies: (string | undefined)[]`?

Reply:
Yes. It's worth it and it's definitely preferable than perpetuating the lie with the type. Changing it.

Reply:
If it causes issues, I'm happy to keep as is :)

Reply:
It shouldn't cause an issue as we are making the type less strict, so anything that was accepted before, will be accepted now.